### ISSUE TITLE
Confusing example in hexdocs on assert_email_delivered_with

### ISSUE THREAD

**Issue by speeddragon** [thoughtbot/bamboo#459]
There is [this example](https://hexdocs.pm/bamboo/Bamboo.Test.html#assert_email_delivered_with/1-example) on `assert_email_delivered_with`:

```
email = new_email(text_body: "I love coffee")
assert_email_delivered_with(email, text_body: ~r/love/) # Will pass
assert_email_delivered_with(email, text_body: ~r/like/) # Will fail
```

But `assert_email_delivered_with` has the arity of 1, and not 2. Is this a bug ?

**Comment by maymillerricci** [comment]
@speeddragon You are correct - great catch! Feel free to open a PR to fix this documentation if you're up for it. Otherwise we will fix this soon. Thanks for reporting!

**Comment by speeddragon** [comment]
Also was missing the `deliver` of the email. Just open the PR.

**Comment by maymillerricci** [comment]
Resolved via #461. Thanks!
